data/pokemon_rule_output.txt

```diff
@@ -0,0 +1,147 @@
+
+
+
+-------Primary Color-------
+['hasPrimaryColor Green', 'hasPrimaryColor Red', 'hasPrimaryColor Blue', 'hasPrimaryColor White', 'hasPrimaryColor Brown', 'hasPrimaryColor Yellow', 'hasPrimaryColor Purple', 'hasPrimaryColor Pink', 'hasPrimaryColor Gray', 'hasPrimaryColor Black']
```

Comment:
Due to my other primary color rule, to avoid confusion please name this predicate "isColor Green".